src/main-process/main.js

```diff
@@ -129,6 +136,26 @@ app.on('ready', () => {
   // integrations
   const integrations = new IntegrationManager(mainWindow);
   integrations.enable();
+
+  // Update tray icon for gnome-shell
+  if(process.platform === 'linux') {

```

Comment:
we have a subdir for platform-specific integrations, currently there is only win32 file, but nothing is preventing you to add linux.js and keep this file more generic

Reply:
fixed